styles.css

```diff
@@ -148,3 +157,119 @@ li {
 .justify-content {
   justify-content: space-between;
 }
+
+.about-work {
+  height: 104px;
+  margin: 79.85px 24px 0 24px;
+}
+
+.line2 {
+  margin: 33px 24px 0 24px;
+}
+
+.about-work h1 {
+  font-size: 40px;
+  line-height: 52px;
+  font-weight: 500;
+  color: #9490ff;
+  font-style: normal;
+}
+
+.background-div {
+  background-image: url("Images/Work-Ellipse.png");
+  background-position: 104% -35%;
+  background-repeat: no-repeat;
+  background-size: cover;
+  min-height: 100vh;
```

Comment:
- [ ] This also is causing an error in the view for the background images I can see that they are slightly not aligned   as the original design.